test/collection_test.js

```diff
@@ -174,6 +174,55 @@ describe("Collection", () => {
     });
   });
 
+  describe("#_setPlugins", () => {
+    var articles;
+
+    beforeEach(() => articles = testCollection());
+
+    it("should check that first argument is a supported type", () => {
+      try {
+        articles._setPlugins("bogus", {});
```

Comment:
OK, so maybe `Collection#setPlugins` and `Collection#plugins` should be public, just like [these](https://github.com/michielbdejong/kinto.js/blob/148-collection-configure/src/collection.js#L72).

The reason to separate `setPlugins` from the constructor is that the collections are cached in index.js, so only one instance of each collection exists on the kinto object. This means that if you call `kinto.collection("articles", options)` twice, the `Collection` constructor is only called the first time, but the last call should (I think) win in setting the collection-level options for both instance references.